en/docs/quick-starts/java.md

```diff
@@ -1,365 +1,426 @@
-# Start enable Authentication for JAVA App
+# Start enable Authentication for SpringBoot App
 
 !!! Tip 
-    [Try Our Sample](https://github.com/wso2/samples-is/tree/master/oidc-samples/spring-boot-app-sample)
+    [Try Our Sample](../java-sample)
     
-< Intro - Guide to enable authentication for a web application -
-references >
 
 ## Overview
- < Explain the use case with diagrams - can be links to concepts > 
+This showcases the capability to integrate your spring-boot application with  WSO2 Identity Server for
+ secure authentication using OpenID Connect standard.
+
+## Register Application
+
+ 1. Access WSO2 Identity Server Developer Portal .
+  
+  2. Go to **Applications** and click **New Application**.
+   
+  3. Click **Show More**
+  
+  4. Open **OIDC Web Application** template.
+   
+  5. Enter a **Name** for the application 
+        
+      This indicates the name of your application. 
+          
+      **Example**: sample-app
+    
+  6. Click **Next**
+  
+  7. Enter **Callback URL**. 
+  
+     The **Callback URL** is the exact location in the service provider's application where a authorization code
+     would be sent. This should be always `{baseUrl}/login/oauth2/code/wso2` .
+         
+        **Example**: http://localhost:8080/login/oauth2/code/wso2
+         
+     If you want configure oidc logout also, then you need to add post-logout-url.  You can add multiple callback
+     urls. In WSO2 Identity Server, post-logout-url can be configured via callback URL.
  
- < Image explaining the scenario>
+        **Example**:  http://localhost:8080/spring-boot-app
+ 
+  8. Click **Next**
+  
+  9. View the application details and Click **Finish**
+  
+  10. Click on **Access** tab and Note the the **Client ID** and **Client Secret** that appears. 
```

Comment:
the **Access** tab and note the **Client ID** and **Client Secret** that appear.